### PR TITLE
About Me layout

### PR DESCRIPTION
**Pull request by anton25360**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -27,18 +28,53 @@ p1{
...
+h2{
+	display: inline-block;
+	font-size: 3vw;
+	margin: 2px;
 }
+
+.wrapperAbout{
+	margin: 0 10%;
+	display: flex;
+	/* flex-direction: row; */
```
Comment (mporam) [inline style.css]: if you dont need this, please remove it from the PR

File: style.css [hunk 2]
```diff
@@ -27,18 +28,53 @@ p1{
...
+	flex-flow: row ;
+}
+
+.wrapperAbout .text{
+	margin: 10px;
+	font-size: 20px;
+	/* border: 2px solid blue; */
```
Comment (mporam) [inline style.css]: same with this

File: style.css [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
 }
 
 body {
 	margin: 0;
 	padding: 0;
 }
 
+@media only screen and (max-width: 420px) {
+	/* For mobile phones: */
+	*{
```
Comment (mporam) [inline style.css]: space after the selector before the bracket: `* {`

File: style.css (hunk 1 above)
Comment (anton25360) [reply style.css]: done

File: style.css (hunk 2 above)
Comment (anton25360) [reply style.css]: done

File: style.css (hunk 3 above)
Comment (anton25360) [reply style.css]: done

**Pull request merged**